lms/static/sass/shared-v2/_variables.scss

```diff
@@ -26,6 +26,7 @@ $lms-border-color: palette(grayscale, back) !default;
 $lms-label-color: palette(grayscale, black) !default;
 $lms-active-color: palette(primary, base) !default;
 $lms-preview-menu-color: #c8c8c8 !default;
+$lms-inactive-color: rgb(94,94,94) !default;
```

Comment:
I wouldn't define rgb values here, however the v2 of styling is being deprecated so I think this is fine in this case - we don't want to waste too much time with these pages because they will soon (hopefully) be converted.